release_notes.md

```diff
@@ -0,0 +1,11 @@
+## What's New
+
+### Enhanced Task Flow in Streamline Section
+
+The **Streamline** section now offers a smoother setup experience with optimized task ordering. Start by configuring indicator extraction &mdash; it's now the first step to help you get off to a powerful start.
+
+- **New First Task**: *Configure Indicator Extraction*  
+   Dive into your setup by prioritizing indicator extraction, now in its most prominent position.
+
+   - **Updated Launch Button**: *Configure Exclude List* is now **Configure Indicator Extraction and Exclusion**  
```

Comment:
@cs-abhishek-shukla , Please replace "Configure Indicator Extraction and Exclusion" with "Manage Indicator Exclusion List"

![image](https://github.com/user-attachments/assets/52bced2d-0345-43e6-b986-2d033eef3714)